### ISSUE TITLE
Site not working

### ISSUE THREAD

**Issue by swapn652** [ammaaraslam/tweetsnap#3]
**Describe the bug**
It's giving the following error

>Application error: a client-side exception has occurred (see the browser console for more information).

**Screenshots**
![err](https://user-images.githubusercontent.com/70851202/175929908-4c744bd7-3e9f-473d-91fa-051dd08c0eda.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Microsoft Edge
 - Version: 103.0.1264.37

**Comment by ammaaraslam** [comment]
Hi there @swapn652. Sorry you had to encounter this issue.
It seems that the error was due to the Tweet's link.
I was able to reproduce this error when I copied the link to the tweet via the **Copy link to Tweet** option in twitter. This is because, as of now, the app is only able to get the _tweet ID_ if you copied the Tweet's link via the url in the address bar.
This is an easy fix and will start right away to fix it.
Thank you @swapn652 for creating this issue and letting us know.
